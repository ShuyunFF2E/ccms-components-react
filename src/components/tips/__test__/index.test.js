import React from 'react';
import { render, mount } from 'enzyme';

import Tips from '../index';

describe('Tips', () => {
	it('renders correctly', () => {
		const wrapper = render(<Tips msg="默认提示信息" />);
		expect(wrapper).toMatchSnapshot();
	});

	it('it support different types', () => {
		['major', 'warning', 'normal'].forEach(type => {
			const wrapper = mount(<Tips msg="提示信息" type={type} />);
			expect(wrapper.find(`.${type}`)).toBeTruthy();
			wrapper.unmount();
		});
	});
});
