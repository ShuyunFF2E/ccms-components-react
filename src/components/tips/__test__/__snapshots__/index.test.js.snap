// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tips renders correctly 1`] = `
<div
  class="normal cloud-tips-container"
>
  默认提示信息
</div>
`;
